python/python/bystro/ancestry/listener.py

```diff
@@ -90,7 +90,7 @@ def handler_fn(publisher: ProgressPublisher, job_data: AncestryJobData) -> Ances
         # simply threading it through for later.
         _reporter = get_progress_reporter(publisher)
 
-        dataset = ds.dataset([job_data.dosage_matrix_path], format="arrow")
+        dataset = ds.dataset(job_data.dosage_matrix_path, format="arrow")
```

Comment:
this is needed to support partitioned feather files (which are folders with arrow feather files in them that share the same schema)